portfolio/src/main/java/com/google/sps/servlets/SentimentAnalysisServlet.java

```diff
@@ -0,0 +1,47 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import com.google.cloud.language.v1.Document;
+import com.google.cloud.language.v1.LanguageServiceClient;
+import com.google.cloud.language.v1.Sentiment;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/sentiment")
+public class SentimentAnalysisServlet extends HttpServlet {
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String mantra = request.getParameter("mantra");
+
+    Document doc =
+        Document.newBuilder().setContent(mantra).setType(Document.Type.PLAIN_TEXT).build();
```

Comment:
When we write more code, it may be helpful to have a class that helps map the 
1. Request -> Document
2. Sentiment score -> Response
3. Handles the Language Service Client for you. e.g. a LanguageService class that already has the LanguageServiceClient created for you, and each time you can just call LanguageService.getSentiment(document) it will return you a float.